### PR TITLE
Drop Python 3.6 support

### PR DESCRIPTION
**Pull request by Laerte**

### CODE REVIEW

**Review by Gallaecio — APPROVED** [review]
:tada:

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -14,13 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11.0-rc.2", "pypy3"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.2", "pypy3.7"]
```
Comment (wRAR) [inline .github/workflows/tests.yml:17]: Can you change 3.11.0-rc.2 to 3.11 so that we don't make another PR for that please?

**Review by wRAR — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Laerte) [reply .github/workflows/tests.yml:17]: I was about to ask about that haha

Comment (Laerte) [reply .github/workflows/tests.yml]: Done!

**Review by wRAR — APPROVED** [review]

**Comment by wRAR** [comment]
Thanks!
